resources/conf/config.toml

```diff
@@ -86,7 +86,7 @@ enabled = false
 serviceUrl = "https://localhost:9443/"
 username="admin"
 password="admin"
-environmentLabels = ["Production and Sandbox"]
+environmentLabels = ["default"]
```

Comment:
Here the default should be `"Production and Sandbox"` after viraj's latest changes I believe
```suggestion
environmentLabels = ["Production and Sandbox"]
```

Reply:
There was a discussion. hence I changed this. I'll revert this back then. I wasn't aware of Viraj's feature.

Reply:
This commit is reverted.